src/test/java/ch/liquidmind/inflection/compiler/util/InflectionCompilerTestUtility.java

```diff
@@ -0,0 +1,120 @@
+package ch.liquidmind.inflection.compiler.util;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+
+import java.io.File;
+import java.net.URLClassLoader;
+import java.nio.file.Path;
+import java.nio.file.attribute.FileAttribute;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import javax.tools.DiagnosticCollector;
+import javax.tools.JavaCompiler;
+import javax.tools.JavaFileObject;
+import javax.tools.StandardJavaFileManager;
+import javax.tools.ToolProvider;
+
+import com.google.common.io.Files;
+
+import __java.nio.file.__Files;
+import ch.liquidmind.inflection.compiler.CompilationJob;
+import ch.liquidmind.inflection.compiler.CompilationJob.CompilationMode;
+import ch.liquidmind.inflection.compiler.InflectionCompiler;
+import ch.liquidmind.inflection.loader.TaxonomyLoader;
+import ch.liquidmind.inflection.test.TestUtility;
+import ch.liquidmind.inflection.test.mock.AbstractFileMock;
+import ch.liquidmind.inflection.test.mock.InflectionFileMock;
+import ch.liquidmind.inflection.test.mock.JavaFileMock;
+
+public final class InflectionCompilerTestUtility
+{
+	public static CompilationJob createCompilationJob( ClassLoader modelClassLoader, InflectionFileMock... inflectionFileMocks )
```

Comment:
removed public method and refactored utility, will be fixed in 285cc09f640caf5ca36192c5318ec767c5a115de